src/.relint.yml

```diff
@@ -53,6 +53,17 @@
   # imports from .all are allowed in all.py; also allow in some modules that need sage.all
   filePattern: '(.*/|)(?!(all|benchmark|dev_tools|parsing|sage_eval))[^/.]*[.](py|pyx|pxi)$'
 
+- name: 'namespace_pkg_all_import_2: Module-level import of .all of a namespace package'
+  hint: |
+    Sage library code should not import sage.PAC.KAGE.all when sage.PAC.KAGE is an implicit
+    namespace package. Type import_statements("SOME_IDENTIFIER") to find a more specific import,
+    and rewrite the import statement as "from sage.PAC.KAGE.MODULE import ..."
+    or "lazy_import('sage.PAC.KAGE.MODULE', '...')".
+  # Keep in sync with above; but for now we ignore sage.{arith,categories}
```

Comment:
You may want to edit the content of `hint:`.

Reply:
What's your suggestion here?

Reply:
Ah, sorry. The hint is already perfect.